src/languageclient/ReloadSchemasRequest.ts

```diff
@@ -0,0 +1,4 @@
+export class ReloadShemasRequest {
```

Comment:
an interface would make more sense here, don't actually need a runtime concept for it:

export interface IReloadSchemasRequest { 
}

Reply:
Do we need anything at all then? Doesn't seem like that interface would get used. Let me try without it and see if things still work.

Reply:
I'm not sure how RPC / serialization / deserialization works here. I was trying to match parameter class name in case it's significant.

Reply:
I'd be very surprised if it did.  TS is a duck-typed language, so we tend to use classes much less.

Reply:
OK, the name doesn't seem to matter, and we don't need that interface or the type. I'll delete it in the next commit.